pages/code-review.md

```diff
@@ -14,28 +14,61 @@ channel!
 
 ## Why reviews?
 
-Code reviews are incredibly important step to the development process. Our goals
-when reviewing code are to:
+Code reviews are incredibly important step to the development process. Our
```

Comment:
How about "Code reviews are an incredibly important part of the development process"?